### DISCUSSION TITLE
Getting error when I run the seed command.

### DISCUSSION THREAD

**Discussion by mylastore** [Prolifode/deno_rest#62] [Q&A]
```
error: Relative import path "expect/mod.ts" not prefixed with / or ./ or ../
    at file:///Users/oscarquinteros/Projects/deno_rest/deps.ts:25:24
[&] [daem] got status of process with pid 10700
[&] [daem] process with pid 10700 exited on its own
[E] [daem] app crashed - waiting for file changes before starting ...
```

**Comment by vicky-gonsalves** [comment]
> ```
> error: Relative import path "expect/mod.ts" not prefixed with / or ./ or ../
>     at file:///Users/oscarquinteros/Projects/deno_rest/deps.ts:25:24
> [&] [daem] got status of process with pid 10700
> [&] [daem] process with pid 10700 exited on its own
> [E] [daem] app crashed - waiting for file changes before starting ...
> ```

This [PR](https://github.com/Prolifode/deno_rest/pull/66) solves the issue
